src/arc/mp/MultiplayerRoom.as

```diff
@@ -142,12 +142,16 @@ package arc.mp
 
         private function onStateButtonClick(event:MouseEvent):void
         {
-            if(room.isPlayer(currentUser))
+            if (room.isPlayer(currentUser))
             {
-                if(currentUser.gameplay.status == Multiplayer.STATUS_LOADED)
+                if (currentUser.gameplay.status == Multiplayer.STATUS_LOADED)
                 {
                     currentUser.gameplay.status = Multiplayer.STATUS_READY;
                     connection.sendCurrentUserStatus(room);
+                }
+                else if (currentUser.gameplay.status == Multiplayer.STATUS_READY)
+                {
+
```

Comment:
Yes, if not it would give the alert to the user to pick a song, even though a song is already picked and they are ready. In this state there's nothing to do currently so it's left blank.